Library/Homebrew/test/rubocops/keg_only_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "rubocops/keg_only"
+
+describe RuboCop::Cop::FormulaAudit::KegOnly do
+  subject(:cop) { described_class.new }
+
+  specify "keg_only_needs_downcasing" do
+    expect_offense(<<~RUBY)
+      class Foo < Formula
+      ^^^^^^^^^^^^^^^^^^^ 'Because' from the keg_only reason should be 'because'.
```

Comment:
Ideally this should point to the `keg_only` line below. I think that means the `node` isn't set correctly. @GauthamGoli could you provide some help here? Thanks!